widget/tabbed_misc/custom_tasklist.lua

```diff
@@ -0,0 +1,58 @@
+local wibox = require('wibox')
+local awful = require('awful')
+local gears = require('gears')
+local beautiful = require('beautiful')
+local dpi = require("beautiful.xresources").apply_dpi
+
+local function tabobj_support(self, c, index, clients)
+	-- Self is the background widget in this context
+	if not c.bling_tabbed then
+		return
+	end
+
+	local group = c.bling_tabbed 
+
+	-- Single item tabbed group's dont get special rendering
+	if #group.clients > 0 then
+
+		local wrapper = wibox.widget({
+			{
+				{
+					-- This is so dumb... but it works so meh
+					{
+						id = "row1",
+						layout = wibox.layout.flex.horizontal,
+					},
+					{
+						id = "row2",
+						layout = wibox.layout.flex.horizontal,
+					},
+					spacing = dpi(2),
+					layout = wibox.layout.fixed.vertical,
+				},
+				halign = "center",
+				valign = "center",
+				id = 'click_role',
+				widget = wibox.container.place,
```

Comment:
I don't think that centering the icons here is a good thing because the user might not want their icons to be centered. So if they just place the imagebox in their background container without anything else, the image will by default not center I think (I experienced that only the icons of tabbed clients would get centered in one possible config)